spinnaker/provider.go

```diff
@@ -1,12 +1,12 @@
 package spinnaker
 
 import (
-   "os"
-   "io/ioutil"
+	"os"
+	"io/ioutil"
```

Comment:
Pretty sure io/ioutil package is deprecated in favor of just io might be worth switching while making changes.